contracts/FreakTeam.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+
+
+enum Role{it, supportIt, intern}
+enum Skill{novice, advanced, competent, profiency, expert}
+
+struct Freak {
+        string name;
+        uint256 startDate;
+        uint256 employeeNumber;
+        Role choiceRole;
+        Skill choiceSkill;
+
+}
+
+
+contract FreakTeam is ERC1155, AccessControl {
+
+    mapping(address => Freak)  public freaks;
+    address[] public freakAccts;
+
+    bytes32 public constant FINANCIAL_ROLE = keccak256("FINANCIAL");
+    bytes32 public constant HR_ROLE = keccak256("HR_ROLE");
+    bytes32 public constant FREAK_ROLE = keccak256("FREAK_ROLE");
+
+    event addedFreak(address _address, string _name, uint256 startDate, uint256 employeeNumber, Role role, Skill skill);
+
+    constructor (address hr, address financial) ERC1155(""){
```

Comment:
Why did you use an empty string for the base constructor?
What is that string used for?

Reply:
at the moment this variable is useless so I deleted it